documentation/assertions/Promise/to-be-fulfilled.md

```diff
@@ -46,13 +44,13 @@ a regular expression, a function, or an object:
 
 
 ```javascript#async:true
```

Comment:
I think we can kill off async:true in a later commit if we update unexpected-markdown to not use it - it is not really needed.